### DISCUSSION TITLE
Handling negative delays between reference and report

### DISCUSSION THREAD

**Discussion by seabbs** [epinowcast/epinowcast#108] [Ideas]
There is some interest in linking events where the report happens before the reference. An example of this is test positivity and symptom onset. 

Potentially this could be handled by changing the indexing of the delay distribution (i.e add a negative `delay_min` which defaults to 0). 

However, I am not sure this really strictly makes sense, especially when considering partially observed data. How do you know if something observed in real-time will eventually convert to a reference? If you don't (which seems likely) then I assume you can't use this information and hence this has no real-time benefit? Or you have to make a censoring adjustment of some kind. 

Regardless there is demand so worth thinking about or at least thinking about documenting a solution that doesn't require model changes (i.e updating the user on the definition of reference and report).

**Comment by adrian-lison** [comment]
This discussion continues in our discourse community [here](https://community.epinowcast.org/t/handling-negative-delays-between-reference-and-report/37).
